src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -29,7 +29,12 @@
           type="button"
           data-toggle="modal"
           data-target="#studentPhotoUploader">
-        Upload/Edit Photo
+        <div class=""
```

Comment:
- `div` in a `button` is not semantically correct. Use `span` instead.
- Remove the `class` attribute instead of having one with empty value.